src/styles/blocks/starsPlusReviews.scss

```diff
@@ -0,0 +1,42 @@
+.starsPlusReviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  line-height: 16px;
+
+  font-size: 10px;
+  margin-bottom: 24px;
+
+  &__prodReviews {
+    color: $main-accent-color;
+    font-size: 10px;
+    width: 53px;
+  }
+}
+
+.stars {
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md?rgh-link-date=2022-03-08T15%3A54%3A52Z)

Reply:
done